### PR TITLE
sci-visualization/gnuplot (and sparc) cleanup

### PR DESCRIPTION
**Pull request by a17r**
@gentoo/sparc @ulm @gentoo/sci

### CODE REVIEW

**Comment by ulm** [comment]
It is entirely the @gentoo/sparc team's decision if they want to drop a package to unstable.

File: app-emacs/gnuplot-mode/gnuplot-mode-0.7.0.ebuild [hunk 1]
```diff
@@ -11,7 +11,7 @@ SRC_URI="https://github.com/bruceravel/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz
 
 LICENSE="GPL-2+"
 SLOT="0"
-KEYWORDS="alpha amd64 arm ~hppa ia64 ppc ppc64 ~s390 sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos"
+KEYWORDS="alpha amd64 arm ~hppa ia64 ppc ppc64 ~s390 ~sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos"
```
Comment (ulm) [inline app-emacs/gnuplot-mode/gnuplot-mode-0.7.0.ebuild]: This is not necessary. sparc is an experimental profile, so the dependency tree need not be complete.
(But again, it's for the sparc team to decide. I would leave the stable keyword in place, though.)

Comment (a17r) [reply app-emacs/gnuplot-mode/gnuplot-mode-0.7.0.ebuild]: I know it is not necessary, but I tried to be nice wrt dep resolution.

**Comment by mattst88** [comment]
sparc@ is okay with destabilizing gnuplot. Thank you for destabilizing revdeps and use masking gnuplot as well!

**Comment by a17r** [comment]
Thanks, done.

**Pull request closed without merging**